scrapy/core/scheduler.py

```diff
@@ -52,12 +54,13 @@ def from_crawler(cls, crawler):
         dupefilter_cls = load_object(settings['DUPEFILTER_CLASS'])
         dupefilter = create_instance(dupefilter_cls, settings, crawler)
         pqclass = load_object(settings['SCHEDULER_PRIORITY_QUEUE'])
+        dpqclass = load_object(settings['SCHEDULER_DELAYED_REQUESTS_PRIORITY_QUEUE'])
         dqclass = load_object(settings['SCHEDULER_DISK_QUEUE'])
         mqclass = load_object(settings['SCHEDULER_MEMORY_QUEUE'])
         logunser = settings.getbool('SCHEDULER_DEBUG')
         return cls(dupefilter, jobdir=job_dir(settings), logunser=logunser,
                    stats=crawler.stats, pqclass=pqclass, dqclass=dqclass,
-                   mqclass=mqclass, crawler=crawler)
+                   mqclass=mqclass, crawler=crawler, dpqclass=dpqclass)
```

Comment:
This can also be an issue for subclasses that do not reimplement `from_crawler`.

A safer approach would be:

```
scheduler = cls(<old parameters>)
scheduler.dpqclass = dpqclass
return scheduler
```